README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Join the chat at https://gitter.im/loklak/loklak](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/loklak/loklak)
 [![Code Climate](https://codeclimate.com/github/fossasia/loklak_search/badges/gpa.svg)](https://codeclimate.com/github/fossasia/loklak_search)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/cf7beeb59bce451f9e02958f301554d8)](https://www.codacy.com/app/fossasia/loklak_search?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fossasia/loklak_search&amp;utm_campaign=Badge_Grade)
```

Comment:
How it is implemented?

Reply:
Following simple format of adding Codacy badge in repo e.g. https://github.com/codacy/node-codacy-coverage/edit/master/README.md, and it is also provided already by Codacy in project's right access account.